src/VisualStudio/Core/Def/LanguageService/AbstractPackage.cs

```diff
@@ -26,12 +29,25 @@ internal IComponentModel ComponentModel
     protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
     {
         await base.InitializeAsync(cancellationToken, progress).ConfigureAwait(true);
+
         await JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
 
         _componentModel_doNotAccessDirectly = (IComponentModel)await GetServiceAsync(typeof(SComponentModel)).ConfigureAwait(true);
         Assumes.Present(_componentModel_doNotAccessDirectly);
     }
 
+    protected override Task OnAfterPackageLoadedAsync(CancellationToken cancellationToken)
+    {
+        // TODO: remove, workaround for https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1985204
```

Comment:
Looks like the issue is now fixed?
[https://devdiv.visualstudio.com/DevDiv/_git/5d307454-5af2-4dca-871c-650bd7faf3cd/pullrequest/536315](https://devdiv.visualstudio.com/DevDiv/_git/5d307454-5af2-4dca-871c-650bd7faf3cd/pullrequest/536315)

Reply:
It isn't fixed as I explained in the comment :(. I need to ping them.

Reply:
The API will be available in the newer Gladstone version.